guides/common/modules/proc_enabling-rh-cloud-and-insights-client-reports-on-hosts.adoc

```diff
@@ -0,0 +1,19 @@
+[id="enabling-rh-cloud-and-insights-client-reports-on-hosts_{context}"]
+= Enabling rh-cloud and insights-client reports on hosts
+
+You can set the `host_registration_insights` parameter to *True* to enable _rh-cloud_ and _insights-client_ reports.
+{Project} will create _rh-cloud_ reports and _insights-client_ will upload reports to the cloud.
+
+Use the following procedure to change the value of `host_registration_insights` parameter.
+
```

Comment:
```suggestion
= Enabling RH Cloud and Insights client reports on hosts

You can enable the Red Hat Insights client on hosts and have the hosts send RH Cloud reports to the {RHCloud}.

```
I would prefer to clean the introduction a bit. You don't have to duplicate the procedure in the intro.

It would also be good to mention what benefits it brings.